evd_ros_core/RobotGrip goal
---
bool status